### DISCUSSION TITLE
flink-connector-mysql-cdc如何实现只读取snapshot中的数据，当读取完snapshot中的数据后自动完成作业，而不是继续一直运行下去

### DISCUSSION THREAD

**Discussion by shengf19951215** [apache/flink-cdc#3023] [Q&A]
How does flink-connector-mysql-cdc read only the data in the snapshot and automatically complete the job when the data in the snapshot is read, instead of continuing to run forever?

**Comment by yangzehan** [comment]
批处理？

**Reply by shengf19951215** [reply]
想要通过修改源码的方式实现，类似于kafka连接器scan.stop.mode配置的效果，比如可以指定读取的offset，读取到这个offset后就自动完成作业，现在我阅读源码发现读取到指定offset可以实现，但是不知道该怎么自动完成

**Reply by lfift** [reply]
https://debezium.io/documentation/reference/1.9/connectors/mysql.html#mysql-property-snapshot-mode
